server/service/src/processors/transfer/invoice/mod.rs

```diff
@@ -271,6 +271,24 @@ fn get_delete_operation(
 
     Ok(Operation::Delete { linked_invoice })
 }
+#[derive(Debug)]
+#[allow(dead_code)] // The "Wrong" variants with fields don't use the fields yet. Be warned allowing dead_code here silences when we never use a particular variant as well.
+enum InvoiceTransferOutput {
```

Comment:
I really like this! Great to get it in debug logs too 🔥 

Future request would be to have put it in a sep PR than for the patch tho, for speed of review 🥲

Reply:
Fair sorry- I did it for the reasons here: https://github.com/msupply-foundation/open-msupply/pull/8798#discussion_r2265484467

Was quite easy to write dud tests accidentally hitting the `store.created_date` check